### ISSUE TITLE
Golden Sun - Dark Dawn: High Resolutions don't apply in the battles

### ISSUE THREAD

**Issue by mhayeck** [Arisotura/melonDS#521]
Hi.  First, I would like to tell that I really love your work with MelonDS, my congratulations and I'm very thankfull to you. I'm not sure, but I think that I found an issue. When I play Golden Sun and enter a battle, the high resolution don't apply to the 3D models, although in the atributes menus they are upscaled alright.

Thanks.

**Comment by Arisotura** [comment]
I think DeSmuME has the same issue. the game captures the video output and applies postprocessing to it in software, which means it's impossible to upscale it (the captured output has to be at the native resolution).

**Comment by mhayeck** [comment]
Hi Arisotura. I've tested it in the DeSmuME and it dosn't have this issue. In that emulator, in the battles, the 3D models are upscaled alright. The sad part is that my PC configuration dosn't run DeSmuME very well... Also, thanks for the quick reply.

**Comment by Arisotura** [comment]
yup, in dual-screen-3D situations, melonDS doesn't apply upscaling yet, because those involve display capture

this is on my TODO list

**Comment by mhayeck** [comment]
I see, nice to know. Thanks! I'm looking forward the next versions of melonDS.

;)

**Comment by BlinksTale** [comment]
Just +1 for this feature request! MelonDS's lightweight nature makes this much more appealing than DeSmuME for Golden Sun in particular.

**Comment by poudink** [comment]
that is in the plans for melonDS 1.0

**Comment by KalaamNozalys** [comment]
Sorry to necro but I just downloaded the 1.0 and seems like this still causes trouble. Or is there a setting somewhere that I have missed ?
Upscaling works wonder outside of combat but in them its only the native resolution.
A shame given how detailed the models actual are when you look at them in the status menu.

**Comment by sandwichwater** [comment]
yea , display capture is still currently unimplemented

**Comment by Richi08** [comment]
guy's?

**Comment by Arisotura** [comment]
I think Generic was on it, but haven't heard of it in a while now...

maybe I should look into it someday

**Comment by RSDuck** [comment]
We don't make our schedules based on when you plan to play a game.

**Comment by Richi08** [comment]
> We don't make our schedules based on when you plan to play a game.

Well that was rude mate!... I never said YOU NEED to schedule anything around me, have some respect when you talk to people in the future! Only keen to get an update on an open issue since 2019!

**Comment by Richi08** [comment]
> I think Generic was on it, but haven't heard of it in a while now...
> 
> maybe I should look into it someday

Yes please do consider it soon! Looking forward to seeing an update on it! all the best & thankyou so much!

**Comment by BlinksTale** [comment]
> > We don't make our schedules based on when you plan to play a game.
> 
> Well that was rude mate!... I never said YOU NEED to schedule anything around me, have some respect when you talk to people in the future! Only keen to get an update on an open issue since 2019!

Oi, Richi - I want this feature as bad as you do, but telling these wonderful volunteer devs that they are rude and that they should “have some respect” is just increasing the chances they lower this feature’s priority or of frustration with ya. 💀 Please 🙏🏻 please! I don’t want this feature cancelled haha

**Comment by mhayeck** [comment]
Even today, MelonDS is one of my favorite's emulators. Good work and thanks for the 1.0 release. I'm still looking forward to play Golden Sun (and the Dragon Quest games) when the time comes.

I really appreciate the work of the team for the community of gamers. Congrats from Brazil!

**Comment by Richi08** [comment]
> > > We don't make our schedules based on when you plan to play a game.
> > 
> > 
> > Well that was rude mate!... I never said YOU NEED to schedule anything around me, have some respect when you talk to people in the future! Only keen to get an update on an open issue since 2019!
> 
> Oi, Richi - I want this feature as bad as you do, but telling these wonderful volunteer devs that they are rude and that they should “have some respect” is just increasing the chances they lower this feature’s priority or of frustration with ya. 💀 Please 🙏🏻 please! I don’t want this feature cancelled haha

regardless, he was rude!

**Comment by Arisotura** [comment]
I think it was this comment

> Any ETA for this happening soon?? the only game this bothers me about is Golden Sun Dark Dawn & really would love this fixed for my next play through. Thanks guy's!

Generic interpreted it as you asking "can you fix this in N weeks/months?" and didn't like that, I guess.

I don't interpret it this way, but I see how one could...

regardless, this is something that's on our TODO list, so it will be fixed... eventually. we can't really grant it will be fixed at a given date.... I can't speak for other team members, but motivation for a personal project isn't something I really control :/

**Comment by Richi08** [comment]
> I think it was this comment
> 
> > Any ETA for this happening soon?? the only game this bothers me about is Golden Sun Dark Dawn & really would love this fixed for my next play through. Thanks guy's!
> 
> Generic interpreted it as you asking "can you fix this in N weeks/months?" and didn't like that, I guess.
> 
> I don't interpret it this way, but I see how one could...
> 
> regardless, this is something that's on our TODO list, so it will be fixed... eventually. we can't really grant it will be fixed at a given date.... I can't speak for other team members, but motivation for a personal project isn't something I really control :/

Thank you so much, i love the recent work on the emulator! I think we all glad its not been dropped yet very much looking forward to it!......Certainly looking forward to my next play through with it upscaled eventually was all i meant, i blocked him anyway for that comment.

**Comment by sandwichwater** [comment]
just my two cents , but blocking and berating one of the developers of the very emulator youre using is a good way to make them want to work on the feature you want even less

**Comment by Richi08** [comment]
> just my two cents , but blocking and berating one of the developers of the very emulator youre using is a good way to make them want to work on the feature you want even less

 Drop it!
